pkg/deploy/strategy/support/lifecycle.go

```diff
@@ -375,6 +375,11 @@ func makeHookPod(hook *deployapi.LifecycleHook, rc *kapi.ReplicationController,
 
 	gracePeriod := int64(10)
 
+	podSecurityContext, err := kapi.Scheme.DeepCopy(rc.Spec.Template.Spec.SecurityContext)
```

Comment:
@pweil- if DC has RC with more than 1 container, which one will you pick for the hook pod? ;-)